storage/innobase/xtrabackup/test/t/history_on_server.sh

```diff
@@ -102,7 +111,7 @@ vlog "Testing incremental based on history name"
 multi_row_insert incremental_sample.test \({101..200},100\)
 
 innobackupex --history=test1 --incremental \
---incremental-history-name=test1 $backup_dir > /dev/null
+--incremental-history-name=test1 --no-lock $backup_dir > /dev/null
```

Comment:
better to add separate command which is full backup?

Reply:
there is some values like lsn, UUID which use need initial backup value. and results are same with increment backup or with full backup.